src/stores/GasPriceStore.js

```diff
@@ -1,28 +1,61 @@
 import { observable, computed } from "mobx";
 import Web3Utils from 'web3-utils';
-import { getGasPrices } from './utils/web3'
+import { fetchGasPrice, fetchGasPriceFromOracle } from './utils/gas'
 
-const GAS_PRICE_SPEED_TYPE = process.env.REACT_APP_GAS_PRICE_SPEED_TYPE || 'fast'
+const HOME_GAS_PRICE_FALLBACK = process.env.REACT_APP_HOME_GAS_PRICE_FALLBACK
+const HOME_GAS_PRICE_ORACLE_URL = process.env.REACT_APP_HOME_GAS_PRICE_ORACLE_URL
+const HOME_GAS_PRICE_SPEED_TYPE = process.env.REACT_APP_HOME_GAS_PRICE_SPEED_TYPE
+const HOME_GAS_PRICE_UPDATE_INTERVAL = process.env.REACT_APP_HOME_GAS_PRICE_UPDATE_INTERVAL
+const FOREIGN_GAS_PRICE_FALLBACK = process.env.REACT_APP_FOREIGN_GAS_PRICE_FALLBACK
+const FOREIGN_GAS_PRICE_ORACLE_URL = process.env.REACT_APP_FOREIGN_GAS_PRICE_ORACLE_URL
+const FOREIGN_GAS_PRICE_SPEED_TYPE = process.env.REACT_APP_FOREIGN_GAS_PRICE_SPEED_TYPE
+const FOREIGN_GAS_PRICE_UPDATE_INTERVAL = process.env.REACT_APP_FOREIGN_GAS_PRICE_UPDATE_INTERVAL
 
 class GasPriceStore {
-  @observable gasPrices = {};
-  gasPricePromise = null;
+  @observable gasPrice = null
+  oracleUrl = null
+  speedType = null
+  updateInterval = null
```

Comment:
I think we should have a default `updateInterval` in case it is not present on `.env` variables, as we did on `bridge-nodejs` otherwise the `setTimeout` function will use the default which is `0`. May be a good value could be 15 seconds?

Reply:
I agree with this, but I think it should be part of a larger work on validating and adding defaults for the environment variables.

Reply:
@fvictorio what if we just set here 15*60*1000 (15 minutes)? We don't need any validation in order to have some default value.

Reply:
Added default gas price update interval to 15 minutes for now. I agree that later we will need to specify a way to validate all `.env` variables and assign defaults